bodhi/server/models.py

```diff
@@ -1339,6 +1433,21 @@ class ModuleBuild(Build):
         'polymorphic_identity': ContentType.module,
     }
 
+    @property
```

Comment:
Can't you use the `['name']`, `['version']`, `['release']` even in the `RpmPackage` and therefore remove this code from `ModuleBuild` and also remove `_get_n_v_r()` in `RpmBuild`?